docs/project-lifecycle.md

```diff
@@ -0,0 +1,122 @@
+phases.
+
+Therefore Hyperledger has adopted a similar lifecycle process as
+follows:
+
+Projects are in one of five possible states:
+
+-   [*Proposal*](#proposal)
+-   [*Incubation*](#incubation)
+-   [*Active*](#active)
+-   [*Promoted Release*](#promoted-release)
+-   [*Deprecated*](#deprecated)
+-   [*End of Life*](#end-of-life)
+
+Projects may not necessarily move through those states in a linear way
+and may go through several iterations.
+
+# Proposal
+
+Project Proposals shall be submitted to the [TSC](https://www.hyperledger.org/about/leadership)
+for review, using [Proposal Template](https://wiki.hyperledger.org/display/TSC/Project+Proposals).
+Proposals that are approved shall enter into an *Incubation* state, unless
+they are of a refactoring nature, in which case they will be turned over
+to the relevant project maintainer(s) to handle as they deem fit.
+
+A Proposal must:
+
+-   have a clear description
+-   have a well-defined scope
+-   identify committed development resources
+-   identify initial maintainers
+-   be vendor neutral
+
+# Incubation
+
+Approved project proposals enter into *Incubation*. For new
+components/modules, a repository will be created under the
+[Hyperledger Github org](https://github.com/hyperledger)
+and optionally in [JIRA](https://jira.hyperledger.org), if requested. New
+features/capabilities should be handled through pull requests labeled
+with tags that identify the project and tag it as
+*incubator* (and will ideally be capable of being enabled/disabled with feature-flags).
+
+Projects in *Incubation* may overlap with one another.
+Entering *Incubation* is meant to be fairly easy to allow for
+community exploration of different ideas.
+
+Once a project qualifies to be declared *Active*, the
+*project*\'s maintainers can then vote to request a graduation
+review by the TSC.
+
+Projects seeking to graduate from *Incubation* must:
+
+-   have fully functional code base
+-   have test coverage commensurate with other *Active*
+    projects
+-   have an active and diverse community of developers
+-   have a history of releases that follow the *Active*
+    release process
+
+Entering *Incubation* does not guarantee that the project will
+eventually get to *Active* state. Projects may never get
+to *Active* state.
+
+The criteria to exit *Incubation* are defined in the
+Incubation Exit Criteria document.
+
+# Active
+
+Projects that have successfully exited the *Incubation* phase
+are in the *Active* phase.
+
+Anyone may propose that a project be deprecated, by submitting a
+rationale and identifying a substitute project/component (if any). The
+maintainers of the project shall vote on such a request and if it
+passes, make that recommendation to the TSC. Members of the community
+that disagree with the request shall make their case before the TSC. The
+TSC shall consider all points of view and render a final decision to
+deprecate or not.
```

Comment:
This paragraph seems out of place. I suggest moving it to the Deprecated section below.

At the same time, I think it would make sense to add a sentence here pointing out that projects in Active status can have Promoted Releases since this has become the main benefit of being in that state.